proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+```
+# Feature name
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+VRHelp items and VRHelpTitle are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very usefull in case your hands are busy with driving.
```

Comment:
After this one sentence it;s good to add oanther one.
"Safety requirements are very strong in automotive, so possibility to use Voice Recognition should be provided for driver all the time"

I introduction we need to concentrate on issue we have to fix.

Reply:
Changed locally.